content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1063,6 +1113,13 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+{{< note >}}
+Since {{< skew currentVersion >}}, when you enable the `JobPodReplacementPolicy`
```

Comment:
same question about the version in other places as well

Reply:
Yeah, let's track the discussion there: https://github.com/kubernetes/website/pull/46808#discussion_r1692964346, and I will update all places if needed.